py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,28 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
```

Comment:
QQ. `format = 'BT[: {format_device}]'` is _NEW!_ sensible default to work with this _NEW!_ format along with _NEW!_ dbus.

The format I made give (new) users either red `BT` or green `BT: Device`. If we use `format = 'BT: [{format_device}|\?show OFF]'` for format, then it kinda goes against py3stautus default formats such as redgreen `DPMS`, redgreen `XSCR`, redgreen `keyboard_locks`, redgreen `module` etc.

There are few modules that acts slightly different... like `do_not_disturb` where it display redgreen `ON`/`OFF` (w/ opposite color... slightly confusing)... instead of something like redgreen `DNT`. I only mention this because if we try for sensible defaults, then people may keep their py3status config only slightly more clean.

Also with new dbus stuffs, you have to run `bluetooth` service. If not, we'll just get red `bluetooth` exception. That wasn't the case before (ie, new behavior now).

Good timing that we start showing errors in `i3bar` instead of `i3-nagbar`. So we now have red `bluetooth`, red `OFF`, and green `BT: Device`. I want to stay with red `bluetooth`, red `BT`, and green `BT: Device`.